packages/frontend/src/components/EditorLayout/index.tsx

```diff
@@ -40,11 +43,43 @@ export default function EditorLayout(): React.ReactElement {
   })
   const flow: IFlow = data?.getFlow
 
+  // for loading demo modal
+  const onFirstLoad = flow?.config?.demoConfig?.onFirstLoad
+  const { onClose } = useDisclosure()
```

Comment:
no-op can remove, ure not using `isOpen` at all

Reply:
actually can get use to closeModal immediately.
```
const {onOpen, onClose, isOpen} = useDisclosure()
useEffect(() => {
   if (onFirstLoad) { onOpen } 
}, [onFirstLoad])
```

Reply:
hmm actually i think i just don't use it baa thks